### ISSUE TITLE
Error: i.DOMFocus is not a function

### ISSUE THREAD

**Issue by yurydelendik** [wasdk/WebAssemblyStudio#349]
https://sentry.io/webassembly-studio/webassembly-studio/issues/586397632/

```
Error: i.DOMFocus is not a function

TypeError: i.DOMFocus is not a function
    at Object.onHide (https://webassembly.studio/dist/main.bundle.js:24:47757)
    at Object.onHide (https://webassembly.studio/dist/monaco-editor.bundle.js:5:198847)
    at e.hide (https://webassembly.studio/dist/monaco-editor.bundle.js:1:1326767)
    at e.hideContextView (https://webassembly.studio/dist/monaco-editor.bundle.js:1:2044469)
    at https://webassembly.studio/dist/monaco-editor.bundle.js:5:198755
    at e.fire (https://webassembly.studio/dist/monaco-editor.bundle.js:1:22164)
    at https://webassembly.studio/dist/monaco-editor.bundle.js:1:433620
    at e.fire (https://webassembly.studio/dist/monaco-editor.bundle.js:1:22164)
    at https://webassembly.studio/dist/monaco-editor.bundle.js:1:68174
    at r (https://webassembly.studio/dist/main.bundle.js:24:7634)
  at onHide (./src/monaco-controller.ts:41:18)
...
(7 additional frame(s) were not displayed)
```

**Comment by yurydelendik** [comment]
https://github.com/wasdk/WebAssemblyStudio/blob/master/src/monaco-controller.ts#L41

**Comment by yurydelendik** [comment]
https://sentry.io/share/issue/2e300d0bcd034d88b0c51e9bea363e5f/

**Comment by yurydelendik** [comment]
Fixed by 04ccbed22fb71255d8e12c1f44eff1652bad5971
